CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 As this project is pre 1.0, breaking changes may happen for minor version bumps. A breaking change will get clearly notified in this log.
 
+## 0.11.0
+
+* Rename `org.stellar.sdk.requests.PathsRequestBuilder` to `org.stellar.sdk.requests.StrictReceivePathsRequestBuilder`
```

Comment:
@tamirms shall we keep `PathsRequestBuilder` as an alias to `StrictReceivePathsRequestBuilder`?

Reply:
I think renaming the class will make it consistent with https://github.com/stellar/java-stellar-sdk/pull/252